### PR TITLE
Add k3sup instructions

### PR DESCRIPTION
**Pull request by alexellis**
Signed-off-by: Alex Ellis (OpenFaaS Ltd) <alexellis2@gmail.com>

## Description

Add [k3sup](https://k3sup.dev/) installation instructions for OpenFaaS

## Motivation and Context

k3sup is a prototype of "faas-cli install" and makes the
experience much easier for newcomers. The longer, manual, and
more detailed instructions are retained in the documentation.

Advanced users are welcome to follow them.

## How Has This Been Tested?

`k3sup` has been tested by the community and in several blog posts on https://blog.alexellis.io/

### CODE REVIEW

File: lab1b.md [hunk 1]
```diff
@@ -205,187 +204,97 @@ NAME                                   STATUS    ROLES     AGE       VERSION
...
-kubectl apply -f https://raw.githubusercontent.com/openfaas/faas-netes/master/namespaces.yml
+k3sup app install openfaas --loadbalancer
 ```
 
-Now add the helm chart repo for the project:
+If you're using a local Kubernetes cluster or a VM, then run:
 
 ```sh
-helm repo add openfaas https://openfaas.github.io/faas-netes/
+k3sup app install openfaas
```
Comment (LucasRoesler) [inline lab1b.md]: I think you could even unify this into the pervious block by using

```
k3sup app install openfaas --loadbalancer=true/false
```
and tell the user to use false when running in a local cluster.  or inversely to use true when deploying to a cloud

Comment (alexellis) [reply lab1b.md]: The default is false

Comment (alexellis) [reply lab1b.md]: Added a note as suggested.

**Comment by alexellis** [comment]
I'll merge this, feedback welcome from anyone that gets to this later on.

**Pull request merged**